.gitattributes

```diff
@@ -0,0 +1,5 @@
+.gitignore export-ignore
+.pylintrc export-ignore
+.gitattributes export-ignore
+.github export-ignore
+*.log export-ignore
```

Comment:
i would remove this line because there are no .log files in the repo.
They get created locally when someone executes fluster or make

Reply:
good cath up, thanks, and done